packages/access-token/test/prepare-test-env-ava.js

```diff
@@ -0,0 +1,7 @@
+import '@endo/init/debug.js';
```

Comment:
why does this file need to be repeated in every package?

Shouldn't `@endo/ses-ava` take care of this? E.g. in a test you can simply import,
```js
import test from '@endo/ses-ava';
```
This is how Ava works. I don't see why ses-ava needs to work so differently.